examples/deprecation-warning/deprecation-warning.cpp

```diff
@@ -12,7 +12,7 @@ int main(int argc, char** argv) {
     }
 
     // Get only the program name from the full path
-    auto pos = filename.find_last_of('/');
+    auto pos = filename.find_last_of('/\\');
```

Comment:
This won't be able to build, invalid syntax

Reply:
Give me some time and I’ll fix it.